akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -159,7 +160,7 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
 
             case e: StreamFrameEvent if !incomingStreams.contains(e.streamId) ⇒
               // if a stream is invalid we will GO_AWAY
-              pushGOAWAY()
+              pushGOAWAY(debug = s"Incoming streamId:${e.streamId} not present in incomingStreams:${incomingStreams}")
```

Comment:
If we push the debug message with the GOAWAY frame, we should describe it in implementation-indepedent terms, so the text should be more like "Received illegal frame for stream $streamId in state ..."